### ISSUE TITLE
Question: Exit logging

### ISSUE THREAD

**Issue by subesokun** [pinojs/pino#1017]
Hi, I need some clarification regarding the use of the `FinalLogger` for exit logging. My project uses Fastify with logging enabled and the Pino documentation states that the `FinalLogger` should be used for [exit logging](https://github.com/pinojs/pino/blob/master/docs/help.md#exit-logging). However, [here](https://github.com/pinojs/pino/blob/master/docs/api.md#pinofinallogger-handler--function--finallogger) it is written that you don't need to care about the finalLogger in case of sync logging, which is to my understanding the default setting of Pino or?

Basically, my question is if I've to care about special (final) exit logging when using Fastify like
```js
const logger = require('pino')()
const fastify = require('fastify')({
  logger
})
process.on('uncaughtException', (error) => {
  logger.error(error, 'uncaught exception occurred')
})
```

Thanks :) Hope this was the right place to ask this question.

**Comment by mcollina** [comment]
You should not be using final with the default settings.

**Comment by subesokun** [comment]
Thanks for the quick answer and clarification :)
